MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+recursive-include aiofiles *.py
+
+include README.rst
+include MANIFEST.in
```

Comment:
Sorry for a comment after merging but `MANIFEST.in` is not needed in the file.
Maybe it doesn't make a harm but no other project has the line IMHO.
I'm pretty sure aio-libs: aiohttp, yarl, multidict and others works pretty well.

Reply:
MANIFEST.in, MANIFEST and setup.py don’t need to be in MANIFEST.in, indeed.